components/vitel_phone/actions/create-call-report/create-call-report.mjs

```diff
@@ -0,0 +1,61 @@
+import app from "../../vitel_phone.app.mjs";
+import utils from "../../common/utils.mjs";
+
+export default {
+  key: "vitel_phone-create-call-report",
+  name: "Create Call Report",
+  description: "Create a call report. [See the documentation](https://www.vitelglobal.com)",
+  type: "action",
+  version: "0.0.1",
+  props: {
+    app,
+    line: {
+      type: "string",
+      label: "Line",
+      description: "Logged in user extension",
+    },
+    startdate: {
+      type: "string",
+      label: "Start Date",
+      description: "Start date for the report",
+    },
+    enddate: {
+      type: "string",
+      label: "End Date",
+      description: "End date for the report",
+    },
```

Comment:
Minor: You could give an example of the date format that the API is expecting. (Below is just an example. I'm not sure what format the Vitel Phone API uses.
```suggestion
    startdate: {
      type: "string",
      label: "Start Date",
      description: "Start date for the report in [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) timestamp format. Example: `2023-01-01T00:00:00.000Z`",
    },
    enddate: {
      type: "string",
      label: "End Date",
      description: "End date for the report in [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) timestamp format. Example: `2023-01-01T00:00:00.000Z`",
    },
```